scipy/integrate/quadpack.py

```diff
@@ -102,9 +102,14 @@ def quad(func, a, b, args=(), full_output=0, epsabs=1.49e-8, epsrel=1.49e-8,
     Other Parameters
     ----------------
     epsabs : float or int, optional
-        Absolute error tolerance.
+        Absolute error tolerance. Default is 1.49e-8. Quad tries to obtain
+        an accuracy of `abs(i-result) <= max(epsabs, epsrel*abs(i))`
+        where `i` = integral of `func` from `a` to `b`, and `result` is the
```

Comment:
same here, `i` is code (a/b/func are parameters, there single backticks are fine)